packages/components/src/internal/query/api.ts

```diff
@@ -565,6 +565,7 @@ export function selectRowsDeprecated(userConfig, caller?): Promise<ISelectRowsRe
                     containerFilter: userConfig.containerFilter ?? getContainerFilter(userConfig.containerPath),
                     includeMetadata: isSelectRowMetadataRequired(userConfig.includeMetadata, columns),
                     includeTotalCount: userConfig.includeTotalCount ?? false, // default to false to improve performance
+                    parameters: userConfig.parameters,
```

Comment:
I don't think this is necessary given the `Object.assign({}, userConfig, ...` above.